localization/sv_SE.inc

```diff
@@ -0,0 +1,5 @@
+<?php
+$labels = array();
+$labels['internal'] = 'Mailet var internt';
+$labels['unencrypted'] = 'Meddelande mottaget över en okrypterad anslutning!';
+?>
```

Comment:
@SimonKagstrom can you validate this translation ?

Reply:
Thank you so much !, feel free to ping or not people to validate other translations